ecosystem/nft-metadata-crawler-parser/src/worker.rs

```diff
@@ -214,6 +218,7 @@ impl Worker {
         token_uri: String,
         last_transaction_version: i32,
         last_transaction_timestamp: chrono::NaiveDateTime,
+        chain_id: i32,
```

Comment:
What's calling this?

Reply:
Nothing yet, will add db checks in following PR

Reply:
Can you add follow up PRs. will make this easier to understand